lib/granite/action/transaction_manager.rb

```diff
@@ -20,6 +20,12 @@ def transaction(trigger_callbacks_for: nil, &block)
           end
         end
 
+        # Adds a block or listener to be executed after finishing the current transaction.
+        # Callbacks are reset after each transaction.
```

Comment:
What is a `listener`? Can you provide a usage example? I'm looking at the specs and still don't understand.

It's also not completely clear why this functionality is needed at all from the pull request description.